src/app.js

```diff
@@ -0,0 +1,29 @@
+import './assets/scss/app.scss';
+import $ from 'cash-dom';
+
+
+export class App {
+  initializeApp() {
+    let self = this;
+
+    $('.load-username').on('click', function (e) {
+      let userName = $('.username.input').val();
+
+      fetch('https://api.github.com/users/' + userName)
+        .then((response)=> {response.json})
```

Comment:
This won't work. First of all, curly braces used with arrow function means that given function has a body, therefore we need to use `return` statement, if we want to return some value from the function. 
Secondly, `.json` is a method, not a property of `response` object.
Thirdly - parentheses around function's parameter are not necessary, as there is only one.

This line should look like this:
`.then(response => response.json())`